src/customerio-cdp.ts

```diff
@@ -62,14 +62,13 @@ export class CustomerIO {
 
   static readonly screen = async (
```

Comment:
Ignore this file change, it has been catered in [another PR ](https://github.com/customerio/customerio-reactnative/pull/314). I rebased the branch to [feature/cdp-milestone-track-identify](https://github.com/customerio/customerio-reactnative/tree/feature/cdp-milestone-track-identify) that caused this change to show here.